gate-keeper/README.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: pattern
+folder: gate-keeper
+permalink: /patterns/gate-keeper/
```

Comment:
Instead of `gate-keeper`, could we use just `gatekeeper`? The title seems to be missing.

Reply:
sure